src/main/src/main/java/org/geoserver/catalog/impl/WMSLayerInfoImpl.java

```diff
@@ -255,4 +267,20 @@ public void setSelectedRemoteFormats(List<String> selectedRemoteFormats) {
     public void setSelectedRemoteStyles(List<String> selectedRemoteStyles) {
         this.selectedRemoteStyles = selectedRemoteStyles;
     }
+
+    public Integer getMinScale() {
+        return minScale;
+    }
+
+    public void setMinScale(Integer minScale) {
+        this.minScale = minScale;
+    }
+
+    public Integer getMaxScale() {
+        return maxScale;
+    }
+
+    public void setMaxScale(Integer maxScale) {
+        this.maxScale = maxScale;
+    }
```

Comment:
Just occurred to me... in all the last changes to this class equals and hashcode have never been updated? Bad! 
Before they were not needed as this class had no extra fields, but now they do...

Reply:
understood, will override and add both hashCode() and equals